### ISSUE TITLE
Project Management Articles Needed

### ISSUE THREAD

**Issue by SeanFCC** [FreeCodeCamp/wiki#667]
The outline below is a preliminary list of Project Management Articles I am writing. I welcome assistance from the community on the articles as well as adding any additional topics to the outline itself.

I also need assistance creating placeholder articles so that the articles can be linked in other articles as needed even prior to them being completed to reduce the need to go back into previous articles to create the links.

##AGILE Project Management – The Basics##
What is Agile? 
The Agile Manifesto 
What is Scrum? 
What is a Scrum Master? 
What is a User Story? 
What is a Sprint? 
Project Scheduling with Agile. 
Stakeholder Management – Why it matters in an Agile world. 
Who is driving this train? - Cooperation in Agile 
Let's do the math – Project Management Equations 
Agile Story Points & The Story Scoreboard 
Burndown and Burnup – Charting the Project 
 
##Agile Project Management – Beyond the Basics##
Project Managers role in an Agile world. 
 XP – The Radicals of Agile 
 Scrum – An In-depth Look 
Creating effective user stories. 
The secrets of effective meetings 
What is Waterfall? 
Agile Vs Waterfall 
The Magic Bullet? 
The role of testing in Agile. 
Requirements – Don't get in your own way 
The 80/20 Rule 
The role of documentation in Agile 
Measuring Velocity in Agile projects 
Gamification in Project Teams – It can be fun!

**Comment by theflametrooper** [comment]
@SeanFCC I think we could maybe include details around projects like trello or jira etc...

**Comment by Rafase282** [comment]
I checked out the sprint one as it was already merged.

**Comment by SeanFCC** [comment]
Thanks for the great suggestion. I'll add an article around tools.

**Comment by alayek** [comment]
@SeanFCC, when you are checking off article milestones; it would be a good idea to link the item to the Wiki article that was created against the request.

**Comment by SeanFCC** [comment]
@alayek Thanks for the great suggestion.

**Comment by theoneinskane** [comment]
I can help with the stakeholder management and root cause analysis one. In my previous job I was the global expert advisor for problem solving in my company (not software though).

**Comment by Aeris777** [comment]
Hi, I am a new contributor. Would it be okay if I write one of these articles? I studied Project Management last year.

**Comment by Rafase282** [comment]
Yes that would be fine. The have to be on the wiki located at the forum.
